submissions/YuliiaDyka/oop-exercise/index.js

```diff
@@ -0,0 +1,60 @@
+class LivingBeing {
+   constructor(species, name, gender, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+   };
+
+   showProperties() {
+      return Object.getOwnPropertyNames(this).map(prop => this[prop]).join('; ');
+   }
+};
+
+class Human extends LivingBeing {
+   constructor(name, gender, saying) {
+      super("Human", name, gender, saying);
+      this.legs = 2;
+      this.hands = 2;
+   };
+};
+
+class Animal extends LivingBeing {
+   constructor(species, name, gender, saying) {
+      super(species, name, gender, saying);
+      this.legs = 4;
```

Comment:
Why `name` is defined in the base class and `legs` in all classes that have legs?